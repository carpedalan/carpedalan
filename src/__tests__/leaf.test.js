import React from 'react';
import { create } from 'react-test-renderer';

import Leaf from '../leaf';

jest.mock('superagent', () => ({
  post: jest.fn().mockReturnThis(),
  get: jest.fn().mockReturnThis(),
}));

describe('test', () => {
  it('should do some stuff', () => {
    const instance = create(<Leaf />);
    expect(instance.toJSON()).toMatchSnapshot();
  });
});
