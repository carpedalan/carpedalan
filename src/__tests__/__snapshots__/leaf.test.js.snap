// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should do some stuff 1`] = `
Array [
  <input
    onChange={[Function]}
    type="file"
  />,
  <input
    data-test="inputField"
    onChange={[Function]}
    value=""
  />,
  <button
    onClick={[Function]}
    type="button"
  >
    submit
  </button>,
]
`;
